vaccineNotifier.js

```diff
@@ -38,9 +38,11 @@ async function checkAvailability() {
 }
 
 function getSlotsForDate(DATE) {
+    let pincodes = PINCODE.split(',');
```

Comment:
The split operation can be done just once, instead of doing it each time for each cron job.

Reply:
Sure, keep splitting and getting dates for each job without any reason. :)